concepts/05 Widgets/Scheduler/077 Time Zone Support.md

```diff
@@ -1,21 +1,11 @@
-Appointments are displayed in the client time zone. The time-zone offset specified in the [startDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate') or [endDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate') is also taken into account (if there is any). You can force a time zone for an individual appointment via its [startDateTimeZone](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDateTimeZone.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDateTimeZone') and [endDateTimeZone](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDateTimeZone.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDateTimeZone') fields or for all appointments in the widget via its [timeZone](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/timeZone.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone') option. **timeZone** takes precedence over **start**/**endDateTimeZone**.
+The **Scheduler** displays appointments according to the client machine's time zone. If you want to override client settings, use the [timeZone](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone) option.
 
-You can use the following formula to calculate an appointment's resulting time-zone offset:
+You can specify an appointment's start/end time in time zones that differ from the **Scheduler**'s time zone. To do this, use the [startDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDateTimeZone) and [endDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDateTimeZone) options in the appointment's data object. These time zones are converted to the **Scheduler**'s time zone for display. Users can view and edit them in the appointment details form if you enable the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/).[allowTimeZoneEditing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowTimeZoneEditing) option.
```

Comment:
```suggestion
You can specify different time zones for an individual appointment. To do this, use the [startDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDateTimeZone) and [endDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDateTimeZone) options in the appointment's data object. When positioning the appointments on the timetable, the **Scheduler** sums up the offsets of individual time zones with that of the **Scheduler**'s time zone. Users can view and edit individual appointments in the appointment details form if you enable the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/).[allowTimeZoneEditing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowTimeZoneEditing) option.
```

Reply:
You can specify different time zones for an individual appointment. To do this, use the [startDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDateTimeZone) and [endDateTimeZone](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDateTimeZone) options in the appointment's data object. When you position appointments in the timetable, the **Scheduler** calculates the offsets of individual time zones and the **Scheduler**'s time zone. Users can view and edit individual appointments in the appointment details form if you enable the [editing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/).[allowTimeZoneEditing](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/editing/#allowTimeZoneEditing) option.